### ISSUE TITLE
Shopping cart page is totally empty

### ISSUE THREAD

**Issue by eitkocat** [magento/magento2#2777]
The mini cart works properly, even when adding items to the cart, the mini cart shows them.

But when going to the shopping cart page, you get an empty page. Even the HTML returned is empty, without <head>, etc.

**Comment by ishakhsuvarov** [comment]
@eitkocat Thank you for reporting this. 
Could you please provide detailed preconditions and steps to reproduce the issue?

**Comment by eitkocat** [comment]
I don't have any. Only thing I did is to install a design template (http://www.venustheme.com/theme/ves-vigoss/). Also, my site is set to Hebrew. Nothing else that I can think of.

**Comment by ishakhsuvarov** [comment]
@eitkocat Please check it again using a fresh Magento install, without any customizations.

**Comment by pavlindrom** [comment]
I don't have any customizations and here's what I get (I have display_errors enabled):

![image](https://cloud.githubusercontent.com/assets/2158645/11996547/17df1872-aa33-11e5-894d-cdcf9d65f798.png)

The class in question doesn't have setShippingRate called in the source code (except for tests), and neither is the shipping_rate ever set in layouts.

**Comment by pavlindrom** [comment]
I was wrong to think it was a pristine environment. Turns out I forgot I installed the extension I worked on. I have narrowed the issue to having a <sequence> inside the etc/module.xml file. I thought that node is for listing dependencies, but now it's pretty clear that it's more about listing the order dependent modules should be loaded in, and the order I listed them in was breaking the cart (and seems to have broken only the cart).

Found confirmation: http://image.slidesharecdn.com/11magento2composerforextensionsdistribution-150310031813-conversion-gate01/95/sergii-shymko-magento-2-composer-for-extensions-distribution-25-638.jpg?cb=1425957575
